.github/workflows/build.yml

```diff
@@ -11,13 +11,11 @@ on:
       dryRun:
         description: 'Dry-Run'
         default: 'true'
+        required: false
```

Comment:
There's a behavior/logic change here. What's the idea here?

Reply:
github action validation in vscode told me that this field is required, so i've added it with false value to be optional.